app/src/main/java/org/oppia/app/splash/SplashActivity.kt

```diff
@@ -0,0 +1,27 @@
+package org.oppia.app.splash
+
+import android.os.Bundle
+import android.view.WindowManager
+import androidx.appcompat.app.AppCompatActivity
+import org.oppia.app.R
+import android.content.res.Configuration
+
+/*
+* This activity acts as a container to SplashFragment
+* There is a background theme set to this activity to display app logo until the app is initialized
+*/
+class SplashActivity : AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_splash)
+
+        getWindow().setFlags(WindowManager.LayoutParams.FLAG_FULLSCREEN, WindowManager.LayoutParams.FLAG_FULLSCREEN);
+
+        val fragment = SplashFragment()
+        val transaction = supportFragmentManager.beginTransaction()
+        transaction.replace(R.id.fragment_container, fragment)
```

Comment:
Small nit: these four lines can be done in one statement. Let's do that instead of splitting it up.